.storybook/stories/wizard/wizard.stories.ts

```diff
@@ -40,7 +40,7 @@ const defaultStory: Story = args => ({
       <clr-wizard-button type="finish">Finish</clr-wizard-button>
 
       <clr-wizard-page *ngFor="let _ of createArray(pageCount); let i = index">
-        <ng-template clrPageTitle>Page {{ i + 1 }}</ng-template>
+        <ng-template clrPageTitle>{{ useLongPageTitles ? 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua' : '' }} Page {{ i + 1 }}</ng-template>
```

Comment:
Why do we have both a separate story for long titles and a parameter for this one? Shouldn't we have just one or the other?

Reply:
Thanks for pointing this out, I removed the parameter as it is no longer needed since it's been pulled into a separate story.

Reply:
Thanks!